src/components/elements/Hero.svelte

```diff
@@ -1,26 +1,26 @@
 <script>
   import RedButton from '../buttons/WhiteButton.svelte'
 
-  export let title
+  export let title = ''
   export let content = ''
-  export let link
-  export let linkText
+  export let link = ''
+  export let linkText = ''
   export let padding = ''
-  export let contentClasses
+  export let contentClasses = ''
   export let titleClasses = 'text-mcswf-gold'
 </script>
 
 <div class={'max-w-[85rem] mx-auto px-4 pt-inherit ' + padding}>
   <!-- Title -->
   <div class="max-w-6xl mx-auto mt-5 text-center">
-    <h1 class="block text-4xl font-bold md:text-5xl lg:text-6xl {titleClasses}">
+    <h1 class="block font-bold text-xl md:text-[40px] {titleClasses}">
       {title}
     </h1>
   </div>
   <!-- End Title -->
 
   <div class="max-w-3xl mx-auto mt-5 text-center">
-    <p id="content" class="text-white {contentClasses}">
+    <p id="content" class="text-white {contentClasses} md:text-[24px]">
```

Comment:
24px is 2xl. Also, I think if you put the text size here, you won't be able override with contentClasses.

Reply:
changed to 2xl and moved so it can be overridden